test/e2e/README.md

```diff
@@ -6,6 +6,13 @@ This assumes you have already built the main project:
 yarn install
 ```
 
+Additionally, you have already installed dependencies for test apps:
+
+```
+yarn install:react
+yarn install:angular
```

Comment:
These have been removed from the project. We should be able to remove this code block.